doc/benchmarking_tutorial.rst

```diff
@@ -1,36 +1,48 @@
-Benchmarking tutorial
+Benchmarking Tutorial
 =====================
+
+.. note:: This is the new benchmarking method only available in ROS Kinetic, onward. Documentation is lacking for the previous benchmarking tutorial for Jade and earlier.
+
+.. note:: To use this benchmarking method, you will need to download and install the ROS Warehouse plugin. Currently this is not available from debians and requires a source install for at least some aspects. For source instructions, see the bottom of [this page](http://moveit.ros.org/install/source/).
```

Comment:
This is not a valid `rst` link. See the `benchmarking package` link in the paragraph below for the correct formatting.